Jenkinsfile

```diff
@@ -1,28 +1,72 @@
 #!groovy
 
-node {
-    def scmVars = checkout scm
+pipeline {
+  agent none
 
-    def dockerRepoName = 'zooniverse/aggregation-for-caesar'
-    def stackName = 'aggregation-caesar'
-
-    def dockerImageName = "${dockerRepoName}:${BRANCH_NAME}"
-    def newImage = null
+  options {
+    disableConcurrentBuilds()
+  }
 
+  stages {
     stage('Build Docker image') {
-        newImage = docker.build(dockerImageName)
-        newImage.push()
-        newImage.push(scmVars.GIT_COMMIT)
+      agent any
+      steps {
+        script {
+          def dockerRepoName = 'zooniverse/aggregation-for-caesar'
+          def dockerImageName = "${dockerRepoName}:${GIT_COMMIT}"
+          def buildArgs = "--build-arg REVISION='${GIT_COMMIT}' ."
```

Comment:
Just looking at the other PR, does this need to use `scmVars.GIT_COMMIT` rather than `GIT_COMMIT`?

Reply:
no - in the declarative version these values are available as ENV vars directly, not from an explicit checkout var result. e.g. https://github.com/zooniverse/caesar/blob/bf65f2c62f29f1f4c397d7adb3646f76582959ec/Jenkinsfile#L17